integration_tests/assets/docker-compose.base.override.yml

```diff
@@ -0,0 +1,12 @@
+version: '3'
+services:
+  sync:
+    depends_on:
+      - auth
+      - dird
+      - db
+      - rabbitmq
+    environment:
+      TARGETS: "dird:9489 auth:9497 db:5432 rabbitmq:5672"
+      TIMEOUT: "${INTEGRATION_TEST_TIMEOUT}"
+
```

Comment:
Don't create another asset, we want to have the less possible assets. Use `all_route`